### PR TITLE
Add Tagline structs

### PR DESCRIPTION
**Pull request by makotech222**
See https://github.com/LemmyNet/lemmy/pull/2548

### CODE REVIEW

File: src/interfaces/source.ts [hunk 1]
```diff
@@ -569,3 +569,11 @@ export class PrivateMessageReport {
   @Expose()
   updated: Option<string>;
 }
+
+export class Tagline {
+  id: number;
+  local_site_id:  number;
+  content: string;
+  published: string;
+  updated: string;
```
Comment (dessalines) [inline src/interfaces/source.ts:578]: updated is not a string, check your back-end struct again.

File: src/interfaces/views.ts [hunk 2]
```diff
@@ -87,6 +88,8 @@ export class SiteView {
   local_site: LocalSite;
   @Type(() => LocalSiteRateLimit)
   local_site_rate_limit: LocalSiteRateLimit;
+  @Type(() => Tagline)
+  taglines: Tagline[];
```
Comment (dessalines) [inline src/interfaces/views.ts:92]: As per another comment I made on your back-end PR, this should be an `Option<Tagline[]>`. There may be no taglines.

**Review by dessalines — CHANGES_REQUESTED** [review]

File: src/interfaces/source.ts (hunk 1 above)
Comment (makotech222) [reply src/interfaces/source.ts]: Fixed, thanks!

**Review by dessalines — APPROVED** [review]

**Comment by dessalines** [comment]
I've just pushed an RC of this to NPM: `0.17.0-rc.52`

**Pull request merged**